app/src/lib/git/log.ts

```diff
@@ -232,7 +268,19 @@ export function parseRawLogWithNumstat(
   for (let i = 0; i < lines.length - 1; i++) {
     const line = lines[i]
     if (line.startsWith(':')) {
-      const status = forceUnwrap('Invalid log output', line.split(' ').at(-1))
+      const lineComponents = line.split(' ')
+      const srcMode = forceUnwrap(
+        'Invalid log output (srcMode)',
+        lineComponents[0].replace(':', '')
```

Comment:
Shouldn't we use `?.` here? If we don't this will throw for `[0]` being undefined instead of the `Invalid log output` error from forceUnwrap
```suggestion
        lineComponents[0]?.replace(':', '')
```

Reply:
That's right! 😱  Nice catch ❤️